### ISSUE TITLE
Doesn't work with rails 4.2.6

### ISSUE THREAD

**Issue by ramyala** [arvindvyas/Country-State-Select#40]
Added to Gemfile:
gem 'country_state_select',  '>= 3.0.2'

bundle install throws an error due to dependency issues on older version of compass-rails. Is there any reason for sticking to 2.0.4?

Fetching gem metadata from https://rubygems.org/
Fetching version metadata from https://rubygems.org/
Fetching dependency metadata from https://rubygems.org/
Resolving dependencies....
Bundler could not find compatible versions for gem "sprockets":
  In snapshot (Gemfile.lock):
    sprockets (= 3.6.0)

  In Gemfile:
    country_state_select (>= 3.0.2) was resolved to 3.0.2, which depends on
      compass-rails (~> 2.0.4) was resolved to 2.0.4, which depends on
        sprockets (< 2.13)

```
haml_coffee_assets was resolved to 1.17.0, which depends on
  sprockets (>= 2.0)

sass-rails (~> 5.0) was resolved to 5.0.4, which depends on
  sprockets (< 4.0, >= 2.8)

sprockets-es6 was resolved to 0.9.0, which depends on
  sprockets (>= 3.0.0)

sprockets-es6 was resolved to 0.9.0, which depends on
  sprockets (>= 3.0.0)

sprockets-es6 was resolved to 0.9.0, which depends on
  sprockets (>= 3.0.0)
```

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

**Comment by ramyala** [comment]
Uploaded a fix at #41

**Comment by arvindvyas** [comment]
@ramyala can you check why build is failing
